pkg/serviceaccount/claims.go

```diff
@@ -166,6 +183,17 @@ func (v *validator) Validate(_ string, public *jwt.Claims, privateObj interface{
 		podUID = podref.UID
 	}
 
+	// Check special 'warnafter' field for projected service account token transition.
+	warnafter := private.Kubernetes.WarnAfter
+	if warnafter != 0 {
+		if now.After(warnafter.Time()) {
+			klog.Warningf("Stale service account token used, service account: %s", public.Subject)
```

Comment:
same comment about verbosity and a TODO to add to audit

Reply:
Done